src/BasicList.jsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemText from '@mui/material/ListItemText';
```

Comment:
you can import all the items above using one line:
import { Box, List, ListItem, ... } from '@mui/material;